app/src/main/java/com/example/fundo/ui/authentication/LoginFragment.kt

```diff
@@ -0,0 +1,122 @@
+package com.example.fundo.ui.authentication
+
+import android.app.Dialog
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import com.example.fundo.R
+import com.example.fundo.databinding.LoginFragmentBinding
+import com.example.fundo.models.User
+import com.example.fundo.services.Auth
+import com.example.fundo.services.Database
+import com.example.fundo.ui.home.HomeActivity
+import com.example.fundo.utils.Utilities
+import com.example.fundo.utils.Validators
+import com.facebook.AccessToken
+import com.facebook.CallbackManager
+import com.facebook.FacebookCallback
+import com.facebook.FacebookException
+import com.facebook.login.LoginResult
+import com.google.firebase.auth.FacebookAuthProvider
+
+class LoginFragment : Fragment(R.layout.login_fragment) {
+    private lateinit var binding:LoginFragmentBinding
```

Comment:
keep binding as nullable var and set null in onDestroyViewnas per fragment lifecycle